src/_P014_SI70xx.ino

```diff
@@ -148,9 +148,16 @@ boolean Plugin_014(uint8_t function, struct EventStruct *event, String& string)
 
     case PLUGIN_INIT:
     {
+      # if FEATURE_I2C_DEVICE_CHECK
+
       if (P014_I2C_ADDRESS == 0) {
```

Comment:
Is it correct to leave out this default check when the I2C device check feature is not enabled?

Reply:
That setting is not used in the constructor `P014_data_struct()`, only during `PLUGIN_READ` it will be used, so no use in updating it here too, except when doing an I2C check.